src/Workspaces/Core/Portable/CodeFixes/FixAllOccurrences/DocumentBasedFixAllProvider.cs

```diff
@@ -93,15 +92,7 @@ public sealed override IEnumerable<FixAllScope> GetSupportedFixAllScopes()
                     return;
 
                 var newDocument = await this.FixAllAsync(fixAllContext, document, documentDiagnostics).ConfigureAwait(false);
-                if (newDocument == null || newDocument == document)
```

Comment:
this code was duplicated in the CodeFix FixAllProvider, and the CodeRefactoring FixAllProvider.  It was been extracted and is run when this code calls into onDocumentFixed.